firefox_puppeteer/ui/windows.py

```diff
@@ -188,7 +188,7 @@ def __eq__(self, other):
 
     @property
     def closed(self):
-        """Returns closed state of the chrome window
+        """Returns closed state of the chrome window.
```

Comment:
Please compare closely! Here you add the final period, while at the reference you remove it. I would suggest to always have a final period.